lbrynet/tests/unit/dht/test_mutinode.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+#
+# This library is free software, distributed under the terms of
+# the GNU Lesser General Public License Version 3, or any later version.
+# See the COPYING file included in this archive
```

Comment:
This test is being licensed under GPL? But the rest of the project is MIT license.

Reply:
I would like to know that too. Why not licence the all thing under GPL?